src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1298,9 +1317,6 @@ private PeriodicShardSyncManager createPeriodicShardSyncManager() {
         private ILeaseRenewer<KinesisClientLease> leaseRenewer;
         @Setter @Accessors(fluent = true)
         private ShardSyncer shardSyncer;
-        @Setter @Accessors(fluent = true)
```

Comment:
Please double check if this Builder is used by any unit test. If yes, make sure those tests won't be affected by this change.

Reply:
Not affected by any unit tests. This was a change made locally in the last commit so it's not exposed to customers yet either, there should be no impact of removing this.